fy22_cecil_b_moore/join_online_in_person.sql

```diff
@@ -0,0 +1,10 @@
+create table longform_joined as(
+
+select cast(ip_address as varchar)ip_address , q1, q2, q3, q4, q5_gender, q6, q7, "usage", frequency, "mode", mode_issues, condition_1, condition_2, condition_3, condition_4, condition_5, condition_6, condition_7, condition_8, condition_9, priorities, ideas
```

Comment:
you can use double colons (i.e. `::`) to cast things more efficiently in SQL.
This `cast(ip_address as varchar)ip_address` is the same as this: `ip_address::varchar as ip_address`